tests/test_validate_environment.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2021 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
```

Comment:
Let us rename to `test_validate_environments` (plural) for analogy with `validate --environments` CLI option.